### PR TITLE
Lt 2566 improve localization

### PR DESCRIPTION
**Pull request by gponomarev-lykke**

### CODE REVIEW

File: src/Chest.Client/Api/ILocalizedValuesApi.cs [hunk 1]
```diff
@@ -40,5 +40,23 @@ public interface ILocalizedValuesApi
...
         Task<GetLocalizedValuesByLocaleResponse> Get(string locale);
+
+        /// <summary>
+        /// Gets all localized values as a flat object. Sorts by key in [asc] order
+        /// </summary>
+        /// <param name="skip"></param>
+        /// <param name="take"></param>
+        /// <returns></returns>
+        [Get("/api/v2/localized-values")]
+        Task<GetAllLocalizedValuesResponse> Get(int skip = default, int take = 0);
```
Comment (tarurar) [inline src/Chest.Client/Api/ILocalizedValuesApi.cs:51]: Default take = 0 looks strange, usually if you are not passing parameters you are getting first 10-20 records.

File: src/Chest/Data/Repositories/LocalizedValuesRepository.cs [hunk 2]
```diff
@@ -164,5 +165,33 @@ public async Task<List<LocalizedValue>> GetAllByKey(string key)
...
             return values;
         }
+
+        public async Task<PaginatedResponse<LocalizedValue>> GetAllAsync(int skip = 0, int take = 0)
+        {
+            await using var context = _contextFactory.CreateDataContext();
+
+            var distinctKeysQuery = context.LocalizedValues
+                .AsNoTracking()
+                .AsQueryable()
```
Comment (tarurar) [inline src/Chest/Data/Repositories/LocalizedValuesRepository.cs:175]: What is the point of converting to `IQueryable`?

File: src/Chest/Models/v2/LocalizedValues/LocalizedValueByKey.cs [hunk 3]
```diff
@@ -0,0 +1,18 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using Chest.Data.Entities;
+
+namespace Chest.Models.v2.LocalizedValues
+{
+    public class LocalizedValueByKey
+    {
+        public string Key { get; set; }
```
Comment (tarurar) [inline src/Chest/Models/v2/LocalizedValues/LocalizedValueByKey.cs:9]: I expect to not have setters if you are constructing object with constructor.

**Review by tarurar — APPROVED** [review]

File: src/Chest.Client/Api/ILocalizedValuesApi.cs (hunk 1 above)
Comment (gponomarev-lykke) [reply src/Chest.Client/Api/ILocalizedValuesApi.cs:51]: Frontend wants to get all the records by default in this task

File: src/Chest/Data/Repositories/LocalizedValuesRepository.cs (hunk 2 above)
Comment (gponomarev-lykke) [reply src/Chest/Data/Repositories/LocalizedValuesRepository.cs:175]: There's some issue with namespaces, so it has to be explicitly cast to IQueryable

File: src/Chest/Models/v2/LocalizedValues/LocalizedValueByKey.cs (hunk 3 above)
Comment (gponomarev-lykke) [reply src/Chest/Models/v2/LocalizedValues/LocalizedValueByKey.cs:9]: Fixed
I left them for Automapper, but I think we will not map LocalizedValueByKeyContract => LocalizedValueByKey, so it's okay

**Pull request merged**